### PR TITLE
add test to build exporter

### PR DESCRIPTION
**Pull request by fondation451**
Issue: https://github.com/Cour-de-cassation/label/issues/690

## Changes
- add test to buildExporter

### CODE REVIEW

File: packages/generic/backend/src/lib/exporter/buildExporter.spec.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      ].map(treatmentModule.generator.generate);
+      await Promise.all(documents.map(documentRepository.insert));
+      await Promise.all(treatments.map(treatmentRepository.insert));
+      const fakeExporterConfig = buildFakeExporterConfig();
+      const exporter = buildExporter(settings, fakeExporterConfig);
+
+      await exporter.exportTreatedDocuments();
+
+      const exportedDocumentIds = fakeExporterConfig.getExportedDocumentIds();
+      const exportedpseudonymizationTexts = fakeExporterConfig.getExportedpseudonymizationTexts();
```
Comment (BenoitSerrano) [inline packages/generic/backend/src/lib/exporter/buildExporter.spec.ts:71]: camelCase ? :)

File: packages/generic/backend/src/lib/exporter/buildExporter.spec.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+      const exportedlabelTreatments = fakeExporterConfig.getExportedlabelTreatments();
```
Comment (BenoitSerrano) [inline packages/generic/backend/src/lib/exporter/buildExporter.spec.ts:72]: camelCase ? :)

Comment (fondation451) [reply packages/generic/backend/src/lib/exporter/buildExporter.spec.ts]: @BenoitSerrano C'est corrigé 
443a992

File: packages/generic/backend/src/lib/exporter/buildExporter.spec.ts (hunk 1 above)
Comment (fondation451) [reply packages/generic/backend/src/lib/exporter/buildExporter.spec.ts]: @BenoitSerrano C'est corrigé 
443a992

**Review by BenoitSerrano — APPROVED** [review]

**Pull request merged**